### PR TITLE
CE-3757: Script for reseting user weekly contributions count

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Do not merge without
- https://github.com/Wikia/app/pull/10530
- https://github.com/Wikia/app/pull/10540

@Wikia/spitfires

### CODE REVIEW

File: maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		$userIds = ( new WikiaSQL() )
+			->SELECT( 'wup_user' )
+			->FROM( 'wikia_user_properties' )
+			->WHERE( 'wup_property' )->EQUAL_TO( 'editcountThisWeek' )
+			->run( $dbw );
+
+		$result = ( new WikiaSQL() )
+			->DELETE( 'wikia_user_properties' )
+			->WHERE( 'wup_property' )->EQUAL_TO( 'editcountThisWeek' )
+			->run( $dbw );
```
Comment (macbre) [inline maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php:36]: How many rows will be deleted by this query? Pinging @drozdo

Comment (lukaszkonieczny) [inline maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php:36]: Examples based on data from last week contributions:

starwars ~ 250
runescape ~ 200
fallout ~ 320
muppet ~ 50
marvel ~ 220
yugioh ~ 170
disney ~ 530
warframe ~ 320
LOL ~ 285
elderscrolls ~ 130

Comment (macbre) [inline maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php:36]: Thanks for the data, It's not tens of thousands of rows, so we should be fine.

File: maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ * On Special:Community we have weekly users rank with most contributions.
+ * We need to reset it after every Sunday
+ */
+
+require_once( dirname( __FILE__ ) . '/../../Maintenance.php' );
+
+/**
+ * Class resetWeeklyUserContributionsCount
+ */
+class resetWeeklyUserContributionsCount extends Maintenance {
```
Comment (Grunny) [inline maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php]: Class names should be UpperCamelCase.

File: maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<?php
+
+use \Wikia\Logger\WikiaLogger;
+
+/**
+ * On Special:Community we have weekly users rank with most contributions.
+ * We need to reset it after every Sunday
+ */
+
+require_once( dirname( __FILE__ ) . '/../../Maintenance.php' );
```
Comment (Grunny) [inline maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php]: Prefer `__DIR__` over `dirname( __FILE__ )`.

File: maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php [hunk 4]
```diff
@@ -0,0 +1,55 @@
+<?php
+
+use \Wikia\Logger\WikiaLogger;
+
+/**
+ * On Special:Community we have weekly users rank with most contributions.
+ * We need to reset it after every Sunday
+ */
+
+require_once( __DIR__ ) . '/../../Maintenance.php' );
```
Comment (Grunny) [inline maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php]: Syntax error. :)

**Comment by Grunny** [comment]
Once the syntax error is resolved, will LGTM.

**Pull request merged**